### PR TITLE
Libz blitz

### PR DESCRIPTION
**Pull request by brson**
No merge.

cc @aturon @dtolnay @rust-lang/libs

### CODE REVIEW

File: _posts/2017-05-05-libz-blitz.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+into the mechanics and goals of this process.
+
+### Rust standards of quality
+
+In all the work on the standard library, we have gained a sense of what makes
+Rust APIs great: principles like "make performance characteristics clear" and
+"use ownership to encapsulate invariants"; consistent naming conventions like
+`mut`, `ref`, `as`, `into`, that let users intuit what they don't know
+from what they do; small details of presentation that add up, like consistently
+documentating possible error cases in a dedicated "Errors" section; and so many
```
Comment (frewsxcv) [inline _posts/2017-05-05-libz-blitz.md]: `s/documentating/documenting`

**Comment by aturon** [comment]
cc @rust-lang/core

File: _posts/2017-05-05-libz-blitz.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+dependency and workflow story across the entire Rust ecosystem, and makes adding
+dependencies a painless and relatively low-risk proposition. The ability to
+share code with ease is a powerful change to the character of, and audience for,
+traditional systems programming tasks.
+
+[Cargo]: https://blog.rust-lang.org/2016/05/05/cargo-pillars.html
+
+Because of Cargo, we can "include batteries" without literally putting them into
+the standard library; pulling in other crates is nearly as easy as using the
+standard library (and will get even easier this year). But to capitalize on this
```
Comment (frewsxcv) [inline _posts/2017-05-05-libz-blitz.md]: Out of curiosity, is "will get even easier this year" referring to something in particular?

File: _posts/2017-05-05-libz-blitz.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+- [Add `#[doc(html_root_url)]`](https://github.com/BurntSushi/byteorder/issues/77)
+
+Beyond `byteorder` we've already worked this process for several other simple
+crates to get a feel for it: [`bitflags`], [`tempdir`], [`flate2`], and
+[`lazy_static`]. Even these, the most basic of crates, have some work left to do
+but you can expect them to have "1.0" releases soon. As the crates under
+evaluation grow in scope, presumably the tasks that arise will grow as well.
+
+[`byteorder`]: https://docs.rs/byteorder
+[`bitflags`]: https://doc.rust-lang.org/tempdir/tempdir/index.html
```
Comment (sebasmagri) [inline _posts/2017-05-05-libz-blitz.md]: Should this point to https://docs.rs/bitflags/?

File: _posts/2017-05-05-libz-blitz.md [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+
+[`rand`]: https://doc.rust-lang.org/rand/rand/index.html
+
+We are currently drawing this list of foundational crates by combining crates.io
+dependency data, various existing curated crate listings, and good old gut
+feeling. The exact list is definitely up for debate, and we hope that the
+broader community will also apply this process completely independently to
+crates the libs team won't have time to discuss.
+
+### How to help
```
Comment (steveklabnik) [inline _posts/2017-05-05-libz-blitz.md]: Maybe consider phrasing this as a question, given the first sentence.

File: _posts/2017-05-05-libz-blitz.md [hunk 5]
```diff
@@ -0,0 +1,356 @@
...
+RFC]). There's plenty of room for more work here, so if you have ideas about
+further improving discoverability on crates.io, [please start a thread]!
+
+[an RFC]: https://github.com/rust-lang/rfcs/pull/1824
+[please start a thread]: https://internals.rust-lang.org/
+
+### Rust standards of community
+
+We have some idea of what goes into publishing exceptional Rust crates, but the
+Rust library team is not the sole authority of Rust API design — the Rust crate
```
Comment (dtolnay) [inline _posts/2017-05-05-libz-blitz.md]: Em dash should be used without spaces.

File: _posts/2017-05-05-libz-blitz.md (hunk 3 above)
Comment (brson) [inline _posts/2017-05-05-libz-blitz.md]: Yes. Thanks.

File: _posts/2017-05-05-libz-blitz.md (hunk 1 above)
Comment (brson) [inline _posts/2017-05-05-libz-blitz.md]: lol

**Pull request merged**